server/webhook_integration_test.go

```diff
@@ -0,0 +1,81 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestSendToWebhook(t *testing.T) {
+	validPayload := &Payload{Username: "mattermod", Text: "test"}
+
+	mattermost := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, "ok")
+	}))
+	defer mattermost.Close()
+
+	r, err := s.sendToWebhook(mattermost.URL, validPayload)
+	if err != nil {
+		t.Fatal(err.Error())
```

Comment:
We use testify as our testing framework. Let's use that here. 

Same applies to everywhere else.